spec/services/workflow_errors_reporter_spec.rb

```diff
@@ -4,58 +4,20 @@
   let(:full_url) do
     'https://sul-lyberservices-test.stanford.edu/workflow/dor/objects/druid:jj925bx9565/workflows/preservationAuditWF/moab-valid'
   end
-  let(:headers) { { 'Content-Type' => 'application/xml' } }
+  let(:headers) { { 'User-Agent' => 'Faraday v0.15.2' } }
   let(:result) { "Invalid moab, validation error...ential version directories." }
-  let(:body) { "<process name='moab-valid' status='error' errorMessage='#{result}'/>" }
+  let(:body) { "<?xml version=\"1.0\"?>\n<process name=\"moab-valid\" status=\"error\" errorMessage=\"#{result}\"/>\n" }
   let(:druid) { 'jj925bx9565' }
   let(:process_name) { 'moab-valid' }
 
   context '.update_workflow' do
-    it '204 response' do
-      allow(Settings).to receive(:workflow_services_url).and_return('https://sul-lyberservices-test.stanford.edu/workflow/')
+    it 'returns true' do
```

Comment:
please add a comment that it returns true because that is what we ALWAYS get from dor-workflow-service gem and reference https://github.com/sul-dlss/dor-workflow-service/issues/50 in the comment.